Index: refpolicypk-1.0/Rules.modular
===================================================================
--- refpolicypk-1.0.orig/Rules.modular
+++ refpolicypk-1.0/Rules.modular
@@ -56,7 +56,7 @@ load: $(instpkg) $(appfiles)
 # created by semanage
 	@echo "Loading configured modules."
 	@$(INSTALL) -d -m 0755 $(policypath) $(dir $(fcpath))
-	$(verbose) $(SEMODULE) -s $(NAME) -i $(modpkgdir)/$(notdir $(base_pkg)) $(foreach mod,$(mod_pkgs),-i $(modpkgdir)/$(mod))
+	$(verbose) $(SEMODULE) -s $(NAME) -i $(modpkgdir)/$(notdir $(base_pkg)) $(foreach mod,$(mod_pkgs),-i $(modpkgdir)/$(mod)) -p $(DESTDIR) -v
 
 ########################################
 #
Index: refpolicypk-1.0/config/appconfig-standard/seusers
===================================================================
--- refpolicypk-1.0.orig/config/appconfig-standard/seusers
+++ refpolicypk-1.0/config/appconfig-standard/seusers
@@ -1,4 +1,4 @@
 root:root
 gdm:system_u
-bighead:unconfined_u
-__default__:user_u
+#bighead:unconfined_u
+__default__:unconfined_u
Index: refpolicypk-1.0/build.conf
===================================================================
--- refpolicypk-1.0.orig/build.conf
+++ refpolicypk-1.0/build.conf
@@ -18,7 +18,7 @@ TYPE = standard
 # If set, this will be used as the policy
 # name.  Otherwise the policy type will be
 # used for the name.
-NAME = refpolicy_monolithic
+NAME = refpolicy
 
 # Distribution
 # Some distributions have portions of policy
